runtime-modules/content/src/lib.rs

```diff
@@ -185,7 +185,7 @@ pub struct ChannelRecord<MemberId, CuratorGroupId, AccountId> {
 
 // Channel alias type for simplification.
 pub type Channel<T> = ChannelRecord<
-    <T as membership::Trait>::MemberId,
+    <T as common::MembershipTypes>::MemberId,
```

Comment:
Maybe I'm not getting it. Shouldn't we use exclusively `MemberId` out of the membership pallet.
Since it is also used in the storage module+